modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -3,28 +3,28 @@
     class="apos-table__cell-field apos-table__cell-field--labels"
     :class="`apos-table__cell-field--${header.name}`"
   >
-    <span v-if="manuallyPublished && item.modified && !item.submitted && item.lastPublishedAt">
+    <span v-if="manuallyPublished && draft.modified && draft.submitted && draft.lastPublishedAt">
```

Comment:
Shouldn't the last part still be `item.lastPublishedAt`? It's indicating a pending draft on a published thing.

Reply:
`lastPublishedAt` is an aspect of the publication status, which apostrophe always records on the draft for certain, so it should come from `draft`.